### PR TITLE
fix(react-native-text-input-mask): refInput typo

### PR DESCRIPTION
**Pull request by hengkx**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [ ] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-native-text-input-mask ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-native-text-input-mask@latest/))
was missing the following properties:
1. mask
1. unmask
1. setMask
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 69dc6f0c24312e69eebe3cdc8bbad2ccfd386a01
</p>

File: types/react-native-text-input-mask/index.d.ts [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 export type onChangeTextCallback = (formatted: string, extracted?: string) => void;
 
-export interface TextInputMaskProps extends ReactNative.ViewProps, ReactNative.TextInputIOSProps, ReactNative.TextInputAndroidProps, ReactNative.AccessibilityProps {
-    refInput?: React.RefObject<ReactNative.TextInput>;
+export interface TextInputMaskProps
+    extends ReactNative.ViewProps,
+        ReactNative.TextInputIOSProps,
+        ReactNative.TextInputAndroidProps,
+        ReactNative.AccessibilityProps {
+    refInput?: (ref: any) => void;
```
Comment (amcasey) [inline types/react-native-text-input-mask/index.d.ts:18]: I'm not familiar with this library, but maybe it's supposed to be `(ref: React.RefObject<ReactNative.TextInput>) => void)`?

https://github.com/react-native-community/react-native-text-input-mask/blob/faf9a4ae6fd4e788396070542f64ffed17db79f1/index.js#L55

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/react-native-text-input-mask/index.d.ts (hunk 1 above)
Comment (hengkx) [reply types/react-native-text-input-mask/index.d.ts:18]: ref is instance.  not React.RefObject.
https://github.com/react-native-community/react-native-text-input-mask/blob/faf9a4ae6fd4e788396070542f64ffed17db79f1/index.js#L53
```
React.ClassAttributes<TextInputMask>.ref?: string | ((instance: TextInputMask | null) => void) | React.RefObject<TextInputMask> | null | undefined
```

Comment (amcasey) [reply types/react-native-text-input-mask/index.d.ts:18]: And it can be of any type?  It's not somehow related to TextInput?

Comment (hengkx) [reply types/react-native-text-input-mask/index.d.ts:18]: Actually TextInput. I tried to write but the error did not know why

Comment (amcasey) [reply types/react-native-text-input-mask/index.d.ts:18]: `refInput?: (ref: ReactNative.TextInput) => void;` compiles for me.  When running the tests locally, I did have to `npm install` in `types/react`, which was somewhat non-obvious.  Maybe give that a shot?

Comment (hengkx) [reply types/react-native-text-input-mask/index.d.ts:18]: I've already used [react-native-masked-text](https://github.com/benhurott/react-native-masked-text) replacement.
It's not convenient to provide error screenshots.
https://github.com/benhurott/react-native-masked-text/blob/master/index.d.ts#L59

**Review by amcasey — APPROVED** [review]
The type can be made more specific in a future PR.

**Comment by hengkx** [comment]
Ready to merge

**Pull request merged**